variables.tf

```diff
@@ -84,4 +84,10 @@ variable "enable_http_application_routing" {
   description = "Enable HTTP Application Routing Addon (forces recreation)."
   type        = bool
   default     = false
-}
\ No newline at end of file
+}
```

Comment:
Could we leave these not changed?

Reply:
Done (my editor automatically append new line at end of file)